GPU/Common/TextureReplacer.cpp

```diff
@@ -235,6 +235,8 @@ bool TextureReplacer::LoadIniValues(IniFile &ini, bool isOverride) {
 
 	bool filenameWarning = false;
 	if (ini.HasSection("hashes")) {
+		aliases_.clear();
```

Comment:
This seems weird, shouldn't we at least clear it even if there's no `hashes` section?  I guess the change here is that if there's an additional ini for a certain region, we now want to delete any of the previous aliases that were presumably global?

-[Unknown]

Reply:
Hm, no, the intention was just to make sure that if LoadIni is called multiple times, it starts from scratch each time. Not even sure if that makes sense though since inserting into it should be idempotent. 

You're right that this line is in the wrong spot anyway.

But LoadIni is called twice on startup right now (which should probably be regarded a bug).

Reply:
Yeah I definitely got confused here. Removing it.